addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -186,7 +186,8 @@ export class HtmlField extends Component {
     }
     updateValue() {
         const value = this.getEditingValue();
-        if (value !== null && value !== this.props.value) {
+        const lastValue = (this.props.value || "").toString();
+        if (value !== null && !(!lastValue && value === "<p><br></p>") && value !== lastValue) {
```

Comment:
Should be validated by the web_editor team. I think they were also working on a fix

Reply:
validated by nby :+1: